src/router/storeSync/storeSync.config.js

```diff
@@ -108,15 +108,7 @@ const storeSyncConfig = [
     new CrossHairParamConfig(),
     new CompareSliderParamConfig(),
     new LayerParamConfig(),
-    new SimpleUrlParamConfig(
-        'featureInfo',
-        'setFeatureInfoPosition',
-        'setFeatureInfoPosition',
-        (store) => store.state.ui.featureInfoPosition,
-        false,
-        String,
-        FeatureInfoPositions.NONE
-    ),
+    new FeatureInfoPositionParamConfig(),
```

Comment:
Does this solve the issue https://jira.swisstopo.ch/browse/PB-358 ? Seems not but if reworking it here maybe have a quick look to see if you can quickly fix it here otherwise this ticket would probably need to wait on this PR to avoid conflict ?

Reply:
was fixed by https://github.com/geoadmin/web-mapviewer/pull/775